ci/jobs/reboot-worker-management.yml.j2

```diff
@@ -7,4 +7,12 @@ jobs:
     config:
       params:
         AWS_ACC: "{{accounts['management']}}"
+  - .: (( inject meta.plan.fetch-credentials ))
+    config:
+      params:
+        USERNAME_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-user"
+        PASSWORD_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-password"
```

Comment:
Why are these here if we're doing `fetch-credentials` ?

Reply:
They are the keys that refer to the username and password values

Reply:
My bad will remove

Reply:
Check with Jim B before merge